src/main/scala/rocket/RocketCore.scala

```diff
@@ -76,7 +76,8 @@ class RocketCustomCSRs(implicit p: Parameters) extends CustomCSRs with HasRocket
     val mask = BigInt(
       tileParams.dcache.get.clockGate.toInt << 0 |
       rocketParams.clockGate.toInt << 1 |
-      rocketParams.clockGate.toInt << 2
+      rocketParams.clockGate.toInt << 2 |
+      1 << 3 // disableSpeculativeICacheRefill
```

Comment:
do we want to hardcode this?

Reply:
I can add a knob for the presence of this chicken bit if you prefer, but my inclination is to make the knob default to having the chicken bit.